docs/literate/intro/speed.jl

```diff
@@ -0,0 +1,123 @@
+# # Speed measurement
+# Here we will compare the speed of plotting UnfoldMakie with MNE (Python) and EEGLAB (MATLAB).
+#
+# Three cases are measured: 
+# - Single topoplot
+# - Topoplot series with 50 topoplots
+# - Topoplott animation with 50 timestamps
+
+using UnfoldMakie
+using TopoPlots
+using PyMNE
+using PythonPlot
+using BenchmarkTools
+using Observables
+using CairoMakie
+using FFMPEG_jll
+
+# Data input 
+dat, positions = TopoPlots.example_data()
+df = UnfoldMakie.eeg_array_to_dataframe(dat[:, :, 1], string.(1:length(positions)));
+
+# # Topoplots
+
+# UnfoldMakie.jl
+@benchmark plot_topoplot(dat[:, 320, 1]; positions = positions)
+
+# UnfoldMakie.jl with DelaunayMesh
+@benchmark plot_topoplot(
+    dat[:, 320, 1];
+    positions = positions,
+    topo_interpolation = (; interpolation = DelaunayMesh()),
+)
+
+# MNE
+posmat = collect(reduce(hcat, [[p[1], p[2]] for p in positions])')
+pypos = Py(posmat).to_numpy()
+pydat = Py(dat[:, 320, 1])
+
+@benchmark begin
+    f = PythonPlot.figure()
+    PyMNE.viz.plot_topomap(
+        pydat,
+        pypos,
+        sphere = 1.1,
+        extrapolate = "box",
+        cmap = "RdBu_r",
+        sensors = false,
+        contours = 6,
+    )
+    f.show()
+end
+
+# # Topoplot series
+# UnfoldMakie.jl
+@benchmark begin
+    plot_topoplotseries(
+        df;
+        bin_num = 50,
+        positions = positions,
+        axis = (; xlabel = "Time windows [s]"),
+    )
+end
+
+# UnfoldMakie.jl with DelaunayMesh
+@benchmark begin
+    plot_topoplotseries(
+        df;
+        bin_num = 50,
+        positions = positions,
+        topo_attributes = (; interpolation = DelaunayMesh()),
+    )
+end
+
+# MNE
+easycap_montage = PyMNE.channels.make_standard_montage("standard_1020")
+ch_names = pyconvert(Vector{String}, easycap_montage.ch_names)[1:64]
+info = PyMNE.create_info(PyList(ch_names), ch_types = "eeg", sfreq = 1)
+info.set_montage(easycap_montage)
+simulated_epochs = PyMNE.EvokedArray(Py(dat[:, :, 1]), info)
+
+@benchmark simulated_epochs.plot_topomap(1:50)
```

Comment:
there is no averaging involved here, probably doesn change much, but a disclamer should be added

Reply:
averaging of what?

Reply:
averaging over time-samples. in unfoldmakie we ask for 50 topoplots, where each is a mean over 4 samples, whereas in plot_topopmap you ask for the first 50 samples to be visualized.

this is all as far as I remember, and we should just make a note of this. I dont expect taking the mean will take that much time to explain the speed advantage of MNE

Reply:
addressed